demosplan/DemosPlanUserBundle/Logic/UserService.php

```diff
@@ -1002,6 +1004,19 @@ public function collectRoleStatistics(array $users): array
         return $roles;
     }
 
+    public function createMasterUserForCustomer(string $userLogin, Customer $customer): User
+    {
+        return $this->userRepository->add([
+            'firstname'    => '',
+            'lastname'     => '',
+            'email'        => $userLogin,
+            'login'        => $userLogin,
+            'password'     => $this->generateRandomString(128, self::HEX_CHARS),
```

Comment:
Why not just `bin2hex(random_bytes(15))`? 128 seems a bit heavy btw.

Reply:
I didn't know about `random_bytes`. The length was chosen to match the current length of the hashed passwords.

Reply:
I didn't know about `random_bytes` and can rework it to use that. I'm not sure about the length though, it was chosen to match the current length of the hashed passwords.

Reply:
I see. I would suggest to simplify it with `random_bytes` to reduce maintenance costs, choosing the length is up to you, it might be even helpful to not use the exact same length to have a visual marker that this is no real password hash, but I will not insist to the length